client/v2/client.go

```diff
@@ -330,7 +330,7 @@ func (bp *batchpoints) SetRetentionPolicy(rp string) {
 
 // Point represents a single data point
 type Point struct {
-	pt models.Point
```

Comment:
The problem is that the v2 client was intended to have points be immutable.  Exporting the underlying Point defeats the purpose of that.

What about adding a function like:
```
NewPointFrom(pt models.Point) {
  return &Point{pt:pt}
}
```